### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by RohanBhattaraiNP**
-test

### CODE REVIEW

File: caltechdata_api/cli.py [hunk 1]
```diff
@@ -401,25 +415,34 @@ def create_record():
...
                     )
                 else:
                     response = caltechdata_write(
-                        existing_data, token, production=True, publish=False
+                        existing_data, token, production=production, publish=False
                     )
                 rec_id = response
-                print(
+                if production == True: 
+
```
Comment (tmorrell) [inline caltechdata_api/cli.py:435]: I would set a variable for the URL based on production. That way we can have just one copy of the message

**Review by tmorrell — APPROVED** [review]
Good to go once message handling is updated

**Comment by tmorrell** [comment]
Will close #39

File: caltechdata_api/cli.py [hunk 2]
```diff
@@ -508,7 +539,7 @@ def edit_record():
             with open(file_name, "r") as file:
                 metadata = json.load(file)
             response = caltechdata_edit(
-                record_id, metadata, token, production=True, publish=False
+                record_id, metadata, token, production=production, publish=False
             )
             if response:
                 print("Metadata edited successfully.")
```
Comment (tmorrell) [inline caltechdata_api/cli.py:545]: You'll also need to update line 554, 558, 594, and 567

**Comment by RohanBhattaraiNP** [comment]
added the function to do the printing, in edit, checked the URL template to appropriate uploads as well.

**Pull request merged**